management/WFCORE-7235_Platform_MBeans_Refresh.adoc

```diff
@@ -0,0 +1,182 @@
+---
+categories:
+- management
+stability-level: default
+issue: https://github.com/wildfly/wildfly-proposals/issues/728
+feature-team:
+ developer: jfdenise
+ sme:
+  - bstansberry
+ outside-perspective:
+  -
+---
+= Platform MBean evolution
+:author:            Jean-Francois Denise
+:email:             jdenise@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFCORE-7235[WFCORE-7235]
+
+=== Related Issues
+
+=== Stability Level
+// Choose the planned stability level for the proposed functionality
+* [ ] Experimental
+
+* [ ] Preview
+
+* [ ] Community
+
+* [x] default
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+* TBD
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE. 
+// Discuss with QE during the Kickoff state to decide this
+* [x] Engineering
+
+* [ ] QE
```

Comment:
Please adapt this to the current template and get rid of all the sections that are now irrelevant, and add any missing ones. The 'contacts' and 'testing by' stuff is now irrelevant.

Reply:
Ok, done.